### PR TITLE
[EDU-3130] feat: add work rules engine pt

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page for Work with Rules Engine in Edge Firewall.

[EDU-3130]: https://aziontech.atlassian.net/browse/EDU-3130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/secure-jornada/editar-edge-firewall/rules-engine.mdx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    - Em **Status Code**, adicione `401`.
+    - Em **Content Type**, adicione o conteúdo e a descrição do cabeçalho. Exemplo: `application/json`.
+    - Em **Content Body**, insira a mensagem que deseja apresentar aos usuários.
+11. Clique no botão **Save**.
+
+---
+
+## Via API
+
+1. Execute a seguinte requisição `POST` para criar uma regra, substituindo `[TOKEN VALUE]` pelo seu [personal token](/pt-br/documentacao/produtos/guias/personal-tokens/) e a variável `<edge_firewall_id>` pelo valor do seu [id do edge firewall](https://api.azion.com/#58376993-e0dd-4b51-9cd3-41d86122728f):
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/secure-jornada/editar-edge-firewall/rules-engine.mdx:43]:
```suggestion
1. Execute a seguinte requisição `POST` para criar uma regra, substituindo `[TOKEN VALUE]` pelo seu [personal token](/pt-br/documentacao/produtos/guias/personal-tokens/) e a variável `<edge_firewall_id>` pelo valor do [ID do edge firewall](https://api.azion.com/#58376993-e0dd-4b51-9cd3-41d86122728f):
```

File: src/content/docs/pt-br/pages/secure-jornada/editar-edge-firewall/rules-engine.mdx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  },
+  "schema_version": 3
+}
+```
+
+3. Aguarde alguns minutos para que as alterações se propaguem.
+
+:::tip
+Consulte a [documentação da API Azion](https://api.azion.com/) e a [especificação OpenAPI](https://github.com/aziontech/azionapi-openapi/) para saber mais sobre todas as funcionalidades disponíveis via API.
+:::
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/secure-jornada/editar-edge-firewall/rules-engine.mdx:121]:
```suggestion
:::

---

import ContributorList from '~/components/ContributorList.astro'

**Contribuidores** <ContributorList>Contributor</ContributorList>
```

**Review by bru-andrade — APPROVED** [review]
Arrasooou

**Review by MarianaAguilera — APPROVED** [review]
Approved with @bru-andrade 's comments

**Pull request merged**